### PR TITLE
[Bug]Fix init issue for layer_norm in sequence_parallel for non-CUDA device.

### PR DESCRIPTION
**Pull request by ys950902**
As sequence_parallel is added in Megatron-DeepSpeed for layernorm, for current implementation, non-CUDA device is using from torch.nn import LayerNorm for layernorm, there is no attr named sequence_parallel, will cause init error for non-CUDA device.

This pr is to fix this issue.

### CODE REVIEW

**Comment by ys950902** [comment]
https://github.com/microsoft/Megatron-DeepSpeed/pull/429 for layernorm is added in this pr.

**Comment by polisettyvarma** [comment]
@ys950902 will this feature work correctly ? have you done any accuracy check for this ?

File: megatron/model/__init__.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
         from .fused_rmsnorm import RMSNorm
     else:
         from .rmsnorm import RMSNorm
-    from torch.nn import LayerNorm
+    from .layernorm import LayerNorm
```
Comment (tjruwase) [inline megatron/model/__init__.py:16]: Can you please share the failure that is being fixed? I have two concerns about this change: 
 1. It is quite subtle since it does not show the connection to sequence-parallelism
 2. It is unclear to me that new `LayerNorm` is equivalent to `torch.nn.LayerNorm` for non sequence-parallel case. 

So, I would like to further understand the problem and explore alternative solutions. Thanks!

**Comment by tjruwase** [comment]
Closing for lack of response. Please re-open if needed.
